internal/component/processor/error.go

```diff
@@ -19,10 +19,6 @@ func MarkErr(part *message.Part, span *tracing.Span, err error) {
 		span = tracing.GetActiveSpan(part)
 	}
 	if span != nil {
-		span.SetTag("error", "true")
-		span.LogKV(
-			"event", "error",
-			"type", err.Error(),
-		)
+		span.RecordError(err, true)
```

Comment:
I'm recording the stacktrace by default but this should be configurable.

What is the best approach to let users configure if want to record stacktrace or not in this case, since this the places where `RecordError` function is used can't reach the `traces` configuration?